VisualPinball.Unity/VisualPinball.Unity.Editor/Import/VpxImportWizard.cs

```diff
@@ -0,0 +1,117 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System;
+using System.IO;
+using VisualPinball.Unity.Editor.Import;
+
+[Serializable]
+public class VpxImportWizard : EditorWindow
+{
+	#region Menu
+	public static VpxImportWizard Window;
+
+	[MenuItem("Visual Pinball/Import Wizard")]
+	public static void Init()
+	{
+		Window = (VpxImportWizard)GetWindow(typeof(VpxImportWizard));
+		Window.autoRepaintOnSceneChange = true;
+		Window.minSize = new Vector2(800, 300);
+		Window.titleContent = new GUIContent("Visual Pinball Import Wizard"); // here we could attach an icon
+		Window.Show();
+	}
+	#endregion Menu
+
+	public void OnGUI()

```

Comment:
I would prefer a little note letting the user know that it's a dev tool and should only be used when "knowing what doing". Otherwise we'll have complaints about table scripts not working when people use the wizard to import and re-export.

Reply:
Sure, I'll add it.